docs/extra.md

```diff
@@ -0,0 +1,93 @@
+To configure Pelorus to monitor your application, you need 3 information:
```

Comment:
This is things we want to add, but I do not know where to put in the documentation

Reply:
Talked to Michal and we will add in Getting Started -> Overview